amqp_broker.go

```diff
@@ -67,13 +67,20 @@ func NewAMQPConnection(host string) (*amqp.Connection, *amqp.Channel) {
 
 // NewAMQPCeleryBroker creates new AMQPCeleryBroker
 func NewAMQPCeleryBroker(host string) *AMQPCeleryBroker {
+	// See http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-default-queue
+	defaultQueueName := "celery"
+	return NewAMQPCeleryBrokerForQueue(host, NewAMQPQueue(defaultQueueName))
+}
+
+// NewAMQPCeleryBrokerForQueue creates new AMQPCeleryBroker
+func NewAMQPCeleryBrokerForQueue(host string, queue *AMQPQueue) *AMQPCeleryBroker {
```

Comment:
I'd prefer to not create another method but I guess its unavoidable for backward compatibility.

Reply:
Since a new method is added, maybe add a `Options` struct for second argument that can be extend in the future.